index.html

```diff
@@ -164,60 +166,40 @@ <h3>About me</h3>
+            <input type="email" id="email" name="email" placeholder="Email Address"  class="email-address"> 
+            <textarea id="message" placeholder="Hey, I'd like to connect to..." maxlength="500"  class="message"></textarea>
+            <!-- <p class="err-msg">Error</p> -->
+            <button type="submit" value="get-in-touch" class="submit_btn">Get in touch</button>     
+            <button type="submit" value="reset-form" class="reset_btn">Reset</button>      
         </form>
     </section>
     <!-- demo -->
-    <div class="showCard">
+     <div class="show-popup">
        
-    </div>
+    </div> 
     
-    <!-- <div class="projectCard">
-        <span class="close__card"><i class="fa fa-bars"></i></span>
-        <div class="feautred_bgcover">
-            
-        </div>
-        <div class="featured_titles">
-            <h3 class="projectTitle">
-                Keeping track of hundreds of components
-            </h3>
-            <div class="demo_links">
-                <a href=""><img src="./img/feauted_imgs/seelive.png" alt="" class="demo_link"></a>
-                <a href=""><img src="./img/feauted_imgs/sourceCode.png" alt="" class="demo_link"></a>
-            </div>
-        </div>
-        <ul class="skill_sets">
-            <li class="skill_set">css</li>
-            <li class="skill_set">css</li>
-        </ul>
-        <p class="featured__description">
-            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea
-        </p>
-    </div> -->
-    <!-- footer -->
+
     <footer>
         <img class="bg3" src="img/bg-3.png" alt="background">
         <span>swanyeelinn000@gmial.com</span>
         <div class="icons2">
-            <a hre="#"><i class="social fa fa-twitter"></i></a>
-            <a hre="#"><i class="social fa fa-linkedin"></i></a>
-            <a hre="#"><i class="social fa fa-medium"></i></a>
-            <a hre="#"><i class="social fa fa-github"></i></a>
-            <a hre="#"><i class="social fa fa-angellist"></i></a>
+            <a hre="https://twitter.com/ritalynn101"><i class="social fa fa-twitter"></i></a>
+            <a hre="https://www.linkedin.com/in/nandar-lynn-061306206/"><i class="social fa fa-linkedin"></i></a>
+            <a hre="https://medium.com/"><i class="social fa fa-medium"></i></a>
+            <a hre="https://github.com/Newt27/Portfolio-setup_and_mobile_first/"><i class="social fa fa-github"></i></a>
+            <a hre="https://www.angellist.com/"><i class="social fa fa-angellist"></i></a>
```

Comment:
Hello @leonlynn1001 & @Newt27,

Great job adding new links to the a tags. Seem like there's typo with the `href` attributes and the links are not working as it should be. Kindly fix them and please remember to add them in the links of the header section too.